core/server/common/src/main/java/alluxio/master/journal/JournalTool.java

```diff
@@ -104,101 +125,34 @@ public static void main(String[] args) throws IOException {
       System.exit(EXIT_SUCCEEDED);
     }
 
-    dumpJournal();
-  }
-
-  private static void dumpJournal() throws IOException {
-    System.out.printf("Dumping journal to %s%n", sOutputDir);
-    Files.createDirectories(Paths.get(sOutputDir));
-    UfsJournal journal =
-        new UfsJournalSystem(getJournalLocation(), 0).createJournal(new NoopMaster(sMaster));
-    try (PrintStream out =
-             new PrintStream(new BufferedOutputStream(new FileOutputStream(sJournalEntryFile)));
-         JournalReader reader = new UfsJournalReader(journal, sStart, true)) {
-      boolean done = false;
-      while (!done && reader.getNextSequenceNumber() < sEnd) {
-        State state = reader.advance();
-        switch (state) {
-          case CHECKPOINT:
-            try (CheckpointInputStream checkpoint = reader.getCheckpoint()) {
-              Path dir = Paths.get(sCheckpointsDir + "-" + reader.getNextSequenceNumber());
-              Files.createDirectories(dir);
-              readCheckpoint(checkpoint, dir);
-            }
-            break;
-          case LOG:
-            JournalEntry entry = reader.getEntry();
-            out.println(ENTRY_SEPARATOR);
-            out.print(entry);
-            break;
-          case DONE:
-            done = true;
-            break;
-          default:
-            throw new RuntimeException("Unknown state: " + state);
-        }
-      }
-    } catch (Exception e) {
-      LOG.error("Failed to read from journal.", e);
+    try {
+      dumpJournal();
+    } catch (Exception exc) {
+      System.err.println(exc);
```

Comment:
Will this be intuitive to use, if run from the command line usually stderr is not tracked?

Reply:
Dumping to std stream now.